src/py/flwr/server/app.py

```diff
@@ -749,3 +789,36 @@ def _add_args_fleet_api(parser: argparse.ArgumentParser) -> None:
         type=int,
         default=1,
     )
+
+    # Fleet API VCE options
+    vce_group = parser.add_argument_group("Fleet API (VCE) server options", "")
+    vce_group.add_argument(
+        "--client-app",
+        help="For example: `client:app` or `project.package.module:wrapper.app`.",
+    )
+    vce_group.add_argument(
+        "--num-supernodes",
+        type=int,
+        help="Number of SuperNodes to register with the SuperLink.",
+    )
+    vce_group.add_argument(
+        "--backend",
+        default="ray",
+        type=str,
+        help="Simulation Backend that processes a ClientApp.",
+    )
+    vce_group.add_argument(
+        "--backend-config",
+        type=str,
+        default='{"client_resources": {"num_cpus":2, "num_gpus":0.0}}',
+        help='A JSON-like dict, e.g. \'{"<key>":<value>, "<another-key>":<value>}\' to '
+        "configure a backend. Values supported in <value> are those included by "
+        "`flwr.common.typing.ConfigsRecordValues`. "
+        "Pay close attention to how the quotes and double quotes are set.",
```

Comment:
Should we remove (or reword) this last sentence to make the comment a bit friendlier?
```suggestion
        help='A JSON-like dict, e.g. \'{"<key>":<value>, "<another-key>":<value>}\' to '
        "configure the backend. Values supported in <value> are those included by "
        "`flwr.common.typing.ConfigsRecordValues`. "
```